saltproc/version.py

```diff
@@ -71,7 +72,7 @@
 
 NAME = "saltproc"
 MAINTAINER = "Oleksandr Yardas"
-MAINTAINER_EMAIL = "arfc@googlegroups.com"
+MAINTAINER_EMAIL = "oyardas2@illinois.edu"
```

Comment:
are you sure you want all messages to go to your personal email? We could make a mailing list.

Reply:
Let's make a mailing list. I'll revert back to the googlegroups email here